teachers_digital_platform/crtool/src/js/components/pages/EfficacyCriterionPage.js

```diff
@@ -0,0 +1,339 @@
+import React from "react";
+
+import C from "../../constants.js"
+
+export default class EfficacyCriterionPage extends React.Component {
+    changeEfficacyRadio(key, checkedValue) {
+        this.props.changeCriterionAnswer(C.EFFICACY_DISTINCTIVE, key, checkedValue);
+        this.checkDistinctiveIsComplete();
+    }   
+
+    changeEfficacyNotes(key, textValue) {        
+        this.props.changeCriterionNotes(C.EFFICACY_DISTINCTIVE, key, textValue);
+        this.checkDistinctiveIsComplete();
+    }  
+
+    efficacyClearLocalstorage() {
+        this.props.clearLocalstorage();
+    }
+
+    checkDistinctiveIsComplete() {
```

Comment:
This check seems a little suspect to me, I'd think you'd be better served having a more definite state for notes and answers that you could loop over.

Reply:
This was here to demonstrate the status of the DistinctiveButton turning to "complete".  Turns out you set the distinctive to complete when ever the user clicks on "Continue To Summary" button.  I went ahead and modified that because it was a small change.

Also there is some code in ...CriterionPage files that can be pulled out and re-used, I am waiting for test coverage before I do that refactor and to see how the FEWD want to work with the components.